app.js

```diff
@@ -315,10 +328,10 @@ app.use(function (err, req, res) {
 process.removeAllListeners('SIGINT')
 
 process.on('SIGINT', function () {
-  debug('Closing clients...')
+  logger.info('Closing clients...')
   gw.close()
     .catch(err => {
-      debug('Error while closing clients', err)
+      logger.error('Error while closing clients', err)
```

Comment:
@ahochsteger How does errors should be logged? I mean can we add error as second parameter and use the first one as a prefix message?